### DISCUSSION TITLE
New Relic integration LOG + APM

### DISCUSSION THREAD

**Discussion by pedro-lucca** [vercel/vercel#8101] [Ideas]
Hi Guys, please, integration with new relic for LOG and APM, we use new relic for everything

**Comment by amyegan** [comment]
Hi @pedro-lucca. This is a great idea that I know some other people have shown interest in this as well. Added to our list of requested features. Thanks for sharing!

**Reply by amyegan** [reply]
A New Relic integration is available now 🎉  [vercel.com/integrations/newrelic](https://vercel.com/integrations/newrelic)
